addon/components/base/bs-modal/dialog.js

```diff
@@ -204,12 +224,26 @@ export default class ModalDialog extends Component {
   }
 
   _click(e) {
+    if (this.ignoreBackdropClick) {
```

Comment:
```suggestion
    if (this.get('ignoreBackdropClick')) {
```

I think we should use `this.get()` here. It shouldn't make a difference for this specific case but it's easier to reason about if we always use `this.get()` unless support for Ember < 3.1 has been dropped.

Reply:
Yes, in theory that's right, but as it's a private property and not expected to be changed to a CP (where `.get()` would be needed for old Ember versions), I think we can accept this and not cause any further work! :wink: